### PR TITLE
Update user-submission.md

### PR DESCRIPTION
**Pull request by andypunt**
We need to make sure people don't use DL/DGs. Also, can you update the screenshots to the latest ones? The screenshots show an old design that we went away from.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/user-submission.md [hunk 1]
```diff
@@ -61,9 +61,9 @@ Delivering user reported messages to a custom mailbox instead of directly to Mic
 
      - **Microsoft (Recommended)**: The user submissions mailbox isn't used (all reported messages go to Microsoft).
 
-     - **Microsoft and a custom mailbox**: In the box that appears, enter the email address of an existing Exchange Online mailbox.
+     - **Microsoft and a custom mailbox**: In the box that appears, enter the email address of an existing Exchange Online mailbox. Distribution groups not allowed.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
     - **Microsoft and a custom mailbox**: In the box that appears, enter the email address of an existing Exchange Online mailbox. Distribution groups are not allowed.
```

File: microsoft-365/security/office-365-security/user-submission.md [hunk 2]
```diff
@@ -61,9 +61,9 @@ Delivering user reported messages to a custom mailbox instead of directly to Mic
...
 
-     - **Custom mailbox**: In the box that appears, enter the email address of an existing Exchange Online mailbox.
+     - **Custom mailbox**: In the box that appears, enter the email address of an existing Exchange Online mailbox. Distribution groups not allowed.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
     - **Custom mailbox**: In the box that appears, enter the email address of an existing Exchange Online mailbox. Distribution groups are not allowed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**